crates/solver/src/settlement_post_processing/mod.rs

```diff
@@ -1,4 +1,12 @@
+use {
+    crate::solver::score_computation::ScoreCalculator,
+    anyhow::{Context, Result},
+    ethcontract::U256,
+    shared::external_prices::ExternalPrices,
+};
+
 pub mod optimize_buffer_usage;
+pub mod optimize_score;
```

Comment:
Originally didn't realize it but these modules can all be imported privately.

Reply:
Fixed